### PR TITLE
update to use new formation.js

### PR DESCRIPTION
**Pull request by accbjt**
## Description
This is to update vets-website with formation.js. This will remove the dependency of JavaScript from uswds. As of right now the only thing we are using is the accordion JavaScript.

To test this we should test different pages that have the accordion and make sure it is still working correctly.

## Testing done
Local Testing

## Acceptance criteria
- [ ] The accordions on the website should still work correctly

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/discharge-wizard/components/InstructionsPage.jsx
Comment (accbjt) [inline src/applications/discharge-wizard/components/InstructionsPage.jsx]: This wasn't working correctly after removing uswds. `this.state.q1` doesn't have an initial value, so this wasn't correct. I think it was working due to the uswds javascript. formation.js doesn't add an event listener to the element if it's a React component or has a previous event listener assigned to it.

File: src/platform/site-wide/index.js
Comment (ncksllvn) [inline src/platform/site-wide/index.js]: Picky but I think we can remove the parentheses here to be more consistent with the rest of our imports

**Review by ncksllvn — COMMENTED** [review]
This is very cool work!! 

1. On this content page, I found that the accordion is functional, but the plus/minus icons all default to the minus icons instead of plus. Once you toggle one, they all look right though.

image

2. Similar issue with multi-selectable accordions. On this page, you can toggle items, but the each icon is always minus until toggled.

image

File: src/applications/discharge-wizard/components/InstructionsPage.jsx
Comment (jbalboni) [reply src/applications/discharge-wizard/components/InstructionsPage.jsx]: Sounds like we need to test out the accordions in any React apps that have them.

**Comment by jbalboni** [comment]
@accbjt One thing that the USWDS has that our Formation js doesn't is the JS that controls the gov site banner at the top of the page. That appears broken now. It doesn't look like this exists in our design system, so maybe add a click handler in the vets-website js to toggle the banner on and off? 

The USWDS js for this is pretty simple: https://github.com/uswds/uswds/blob/develop/src/js/components/banner.js

**Comment by accbjt** [comment]
@jbalboni thanks that was a good find. Should we just add the banner js formation?

@ncksllvn thanks I will look into the accordion icon issue.

**Comment by accbjt** [comment]
@ncksllvn looks like the accordion button is missing the arias-expanded attribute. I will add some JS to handle this situation. @jbalboni @ncksllvn do you think it's good that JS handles things that are left out in markup? In a way I like it because it makes it more flexible but on the other hand it doesn't promote conformity with writing our markup.

**Comment by jbalboni** [comment]
@accbjt I don't have a strong opinion either way. Since USWDS was handling it, we should probably handle it, too.

**Comment by jbalboni** [comment]
@accbjt Is this ready for me to review? Wasn't sure if you had all the accordion updates from formation in here yet.

**Comment by accbjt** [comment]
Not yet. I need to get the sidebar fix in first.

**Comment by accbjt** [comment]
@jbalboni @rianfowler @ncksllvn This is ready for review again. We integrated the accordion and also the sidenav. Those 2 have been moved over to formation. Please test both of these components and let me know if you find any bugs.

**Review by ncksllvn — APPROVED** [review]
I think @jbalboni and/or @rianfowler should review as well, but this LGTM! Nice work!

**Comment by accbjt** [comment]
@jbalboni @rianfowler can you take a quick look at this before merging. It would be nice to get more eyes on this.

**Comment by jbalboni** [comment]
@accbjt If I run this locally and go to /sign-in-faq, there's a JS error and the accordions don't work.

**Comment by accbjt** [comment]
@jbalboni checked this out and it was actually a bug in vets-website. Once I pulled from master it got fixed. It should be good now. This page shouldn't have had the sidenav which is why it was causing an error.

**Comment by jbalboni** [comment]
@accbjt I'm still seeing the sidebar and the error and I've got the latest from your branch.

**Comment by accbjt** [comment]
@jbalboni did you do a `$ yarn watch --pull-drupal`

**Comment by jbalboni** [comment]
@accbjt I did. In any case, this page is broken because of CMS stuff, and it's not a prod issue, so I'll bring it up with them.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**
